src/features/DisplayControls/Diagnosis/styles/Diagnosis.scss

```diff
@@ -0,0 +1,3 @@
+.no-dignosis-message {
+  padding-top: 0.625rem;
```

Comment:
it's always better to use px or percentage for margin and padding depending on the use case

Reply:
updated to px